qa/suites/upgrade/quincy-p2p/quincy-p2p-parallel/point-to-point-upgrade.yaml

```diff
@@ -0,0 +1,178 @@
+    - osd_map_max_advance
+    - wrongly marked
+    - FS_DEGRADED
+    - POOL_APP_NOT_ENABLED
+    - CACHE_POOL_NO_HIT_SET
+    - POOL_FULL
+    - SMALLER_PG
+    - pool\(s\) full
+    - OSD_DOWN
+    - missing hit_sets
+    - CACHE_POOL_NEAR_FULL
+    - PG_AVAILABILITY
+    - PG_DEGRADED
+    - application not enabled
+    - cache pools at or near target size
+    - filesystem is degraded
+    - OBJECT_MISPLACED
+    ### ref: https://tracker.ceph.com/issues/40251
+    #removed see ^ - failed to encode map
+
+    fs: xfs
+
+    conf:
+      global:
+        mon_warn_on_pool_no_app: false
+        mon_mds_skip_sanity: true
+      mon:
+        mon debug unsafe allow tier with nonempty snaps: true
+      osd:
+        osd map max advance: 1000
+        osd_class_default_list: "*"
+        osd_class_load_list: "*"
+      client:
+        rgw_crypt_require_ssl: false
+        rgw crypt s3 kms backend: testing
+        rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo= testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+roles:
+- - mon.a
+  - mds.a
+  - osd.0
+  - osd.1
+  - osd.2
+  - mgr.x
+- - mon.b
+  - mon.c
+  - osd.3
+  - osd.4
+  - osd.5
+  - client.0
+- - client.1
+openstack:
+- volumes: # attached to each instance
+    count: 3
+    size: 30 # GB
+tasks:
+- print: "****  done quincy about to install v17.2.0 "
+- install:
+    tag: v17.2.0
+    # line below can be removed its from jewel test
+    #exclude_packages: ['ceph-mgr','libcephfs2','libcephfs-devel','libcephfs-dev', 'librgw2']
+- print: "**** done v17.2.0 install"
+- ceph:
+   fs: xfs
+   add_osds_to_crush: true
+- print: "**** done ceph xfs"
+- sequential:
+   - workload
+- print: "**** done workload v17.2.0"
+
+
+#######  upgrade to v17.2.3
+- install.upgrade:
+    #exclude_packages: ['ceph-mgr','libcephfs2','libcephfs-devel','libcephfs-dev']
+    mon.a:
+      tag: v17.2.3
+    mon.b:
+      tag: v17.2.3
+- parallel:
+   - workload_quincy
+   - upgrade-sequence_quincy
+- print: "**** done parallel quincy v17.2.3"
+
+####  upgrade to latest quincy
+- install.upgrade:
+    mon.a:
+    mon.b:
+- parallel:
+   - workload_quincy
+   - upgrade-sequence_quincy
+- print: "**** done parallel quincy branch"
+
+#######################
+workload:
+   sequential:
+   - workunit:
+       clients:
+         client.0:
+           - suites/blogbench.sh
+workload_quincy:
+   full_sequential:
+   - workunit:
+       branch: quincy
+       #tag: v17.2.0
+       clients:
+         client.1:
+         - rados/test.sh
+         - cls
+       env:
+         CLS_RBD_GTEST_FILTER: '*:-TestClsRbd.snapshots_namespaces'
+   - print: "**** done rados/test.sh &  cls workload_quincy"
+   - sequential:
+     - rgw: [client.0]
+     - print: "**** done rgw workload_quincy"
+     - s3tests:
+         client.0:
+           force-branch: ceph-quincy
+           rgw_server: client.0
+           scan_for_encryption_keys: false
+     - print: "**** done s3tests workload_quincy"
```

Comment:
The s3tests are in this `workload_quincy` task, which run for 17.2.3 and the latest quincy. Only the latest quincy has https://github.com/ceph/ceph/pull/46467, which is the PR with the s3test filter which is failing on 17.2.3. I think this particular test is sensitive to the quincy point version, so we need to run 17.2.3 s3tests for 17.2.3, and latest s3tests for quincy latest.

Reply:
thanks @ljflores! we don't track point releases in s3-tests, but i've created a `ceph-quincy-17.2.3` branch without that PR that we can use here

Reply:
@cbodley great! I also just pushed a test commit (can be removed) to see whether the parallel p2p test will complete if we run the tests against specific point versions. But we can also use `ceph-quincy-17.2.3`.

Reply:
@cbodley got the same test failure with my commit:
http://pulpito.front.sepia.ceph.com/lflores-2022-09-20_21:02:09-upgrade:quincy-p2p:quincy-p2p-parallel-quincy-distro-default-smithi/7039482/

Want to try using `ceph-quincy-17.2.3`? I checked ceph/ceph and ceph/ceph-ci for this branch, and wasn't able to find it. So let me know what we can do to use it.

Reply:
@cbodley now that we are on the upgrade path of v17.2.0 --> v17.2.5 --> latest quincy, the s3 tests are behaving! Thanks for your help in this.